backend/routes/api/current.js

```diff
@@ -6,9 +6,11 @@ const { setTokenCookie } = require("../../utils/auth");
 const { User } = require("../../db/models");
 
 const { validateLogin } = require("../../utils/validation");
+const booking = require("../../db/models/booking");
```

Comment:
The more conventional way of doing this import would be to add Booking to your other models import statement on line 6